openrouteservice/src/test/java/org/heigit/ors/apitests/isochrones/ResultTest.java

```diff
@@ -478,4 +479,23 @@ void testCompleteMetadata() {
                 .body("metadata.containsKey('system_message')", is(true))
                 .statusCode(200);
     }
+
+    @Disabled
+    @Test
+    void testAcceptsTimeParameter() {
```

Comment:
AFAIU the purpose of this test is to assess whether providing the time parameter in the case traffic data is enabled yields a valid response, and does not care about the actual content of the response. As such, it might be better harbored under `ParamsTest`, what do you think?

Reply:
I don't know. Looking through existing tests, the distinction between Params and Results seems arbitrary. I don't mind whether this test is here or there. I can put it where you wish.

Reply:
Thanks, maybe @jlink has some thoughts on this?

Reply:
Maybe there exists a better criterion than Param/Result for chunking the tests. What I do know is that many of the ResultTest classes have way too many tests to find my way around.

Reply:
Not sure what the best strategy is. As long as the test names are clear and descriptive enough I don't mind having the tests aggregated in classes grouped by endpoints. For now I would suggest we try to stick to what is already there and try to be consequent when introducing new stuff.  FWIU the distinction between `Param/Result` tests is that the former ones validate whether certain parameters yield meaningful response while the latter one actually deals with the correctness of the results. But of course, there is no clear distinction, and each one of use might have a slightly different perception where the borderline is. From my point of view it makes sense to move the test unless someone objects.

Reply:
O.K. I have moved the test into ParamsTests and created an issue on the cleanup project board.